schema/alpha/artifacts/model.json

```diff
@@ -1,7 +1,86 @@
-
 {
-
     "$id": "http://github.com/kubeflow/metadata/schema/alpha/artifacts/model.json",
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "description": "TODO: Define the metadata about a model."
+    "examples": [{
```

Comment:
Is this something an end-user is expected to craft and submit?

Reply:
End users are not expected to submit these type definitions, but the platform engineers may need to define them and register them if they want to have types not listed here.

This PR adds these types for `kubeflow.org` so that most kubeflow users can directly save/retrieve data of these types. But if a company wants to customized their own `model` or `data_set` etc, they can define their own schemas.